<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2608</width>
    <height>1927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="temps_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TEMPS :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="robot_tombe_button">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>613</y>
     <width>460</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Robot Tombé</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_actuel_text">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>511</y>
     <width>454</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Score actuel :</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_final_label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>1520</y>
     <width>241</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_final_label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>1520</y>
     <width>371</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Score final:</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_actuel_label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maison_tombe_plus">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>1281</y>
     <width>150</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="perte_premiere_vente_label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>1410</y>
     <width>241</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="perte_premierevente_text">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>1409</y>
     <width>741</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Perte premiere vente :</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_left_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>551</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 72pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Nombre vendu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1970</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Prix actuel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maison_tombe_moins">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>1281</y>
     <width>150</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arbre_parc_moins">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>1280</y>
     <width>150</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arbre_parc_plus">
   <property name="geometry">
    <rect>
     <x>736</x>
     <y>1281</y>
     <width>150</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>1160</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Maison tombée:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>1160</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Arbre Parc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="n_maison_tombe_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>1184</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="n_arbre_parc_label">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>1190</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>160</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: green
</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset_button">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>60</y>
     <width>371</width>
     <height>1551</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="arbre_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(200, 152, 38);
</string>
      </property>
      <property name="text">
       <string>Arbre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cereal_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 250, 175);</string>
      </property>
      <property name="text">
       <string>Céreal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="eau_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Eau</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="farine_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Farine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="planche_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string>Planche</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="papier_toilette_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));
background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
      </property>
      <property name="text">
       <string>Papier cul</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="biere_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.770492, y1:0.447, x2:0.77, y2:0.206579, stop:0 rgba(189, 151, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>Biere</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pain_button">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(212, 188, 65);</string>
      </property>
      <property name="text">
       <string>Pain</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>-90</y>
     <width>241</width>
     <height>1661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_arbre">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_cereal">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_eau">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_farine">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_planche">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_papier_toilette">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_biere">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="n_pain">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="prix_planche_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>876</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_arbre_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>135</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_biere_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>1246</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_pain_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>1431</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_farine_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>690</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_eau_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>505</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_papier_toilette_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>1061</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_cereal_label">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>320</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
